CO2Trade-Login-Register/Service/EntityUsersService.cs

```diff
@@ -104,4 +106,45 @@ public async Task<APIResponse> MyProjects(string idEntityUser)
             return _response;
         }
     }
+
+    public async Task<APIResponse> GetUser(string userId)
+    {
+        try
+        {
+            EntityUser user = await _entityUserRepo.GetUser(userId);
+
+            if (user == null)
+            {
+                _response.IsSuccess = false;
+                _response.StatusCode = HttpStatusCode.BadRequest;
+                _response.ErrorMessage.Add("User not found");
+                return _response;
+            }
+
+            _response.IsSuccess = true;
+            _response.StatusCode = HttpStatusCode.OK;
+
+            EntityUserDTO userDto = new EntityUserDTO()
+            {
+                UserName = user.UserName,
+                BusinessName = user.BusinessName,
+                Address = user.Address,
+                Description = user.Description,
+                PhoneNumber = user.PhoneNumber,
+                CO2Measure = user.CO2Measure,
+                Rol = user.Rol.Description,
+                EntityType = user.EntityType.Description
+        };
+            _response.Result = userDto;
```

Comment:
podemos reducir estas lineas utilizando el mapper 

```suggestion
        _response.Result = _mapper.Map<EntityUserDTO>(entityUser);

```